app/jobs/update_provider_id_job.rb

```diff
@@ -0,0 +1,22 @@
+class UpdateProviderIdJob < ActiveJob::Base
+  queue_as :lupo_transfer
+
+  # retry_on ActiveRecord::RecordNotFound, wait: 10.seconds, attempts: 3
+  # retry_on Faraday::TimeoutError, wait: 10.minutes, attempts: 3
+
+  # discard_on ActiveJob::DeserializationError
+
+  def perform(doi_id, options = {})
+    doi = Doi.where(doi: doi_id).first
+
+    if doi.present? && options[:target_id].present?
+      doi.__elasticsearch__.index_document
+
+      Rails.logger.info "[Transfer] Transferred DOI #{doi.doi}."
```

Comment:
Again, I thought we are not transferring DOIs, only clients.

Reply:
languaged has been chaged